README.md

```diff
@@ -7,19 +7,22 @@ This service provides information about scala library dependencies across the pl
 Used by the catalogue-frontend service deployments page.
 
 #### How it works
-The service gathers dependency information from two sources:
+The service gathers the following information:
 
-* SBT build files read from github
+* Dependencies from SBT build files read from github
   * Covers all HMRC repos, including services, libraries, prototypes
-  * Limited to a whitelist of dependencies, doesnt cover transitive
-  * Only shows the latest dependencies from master branch
-  * Shows if dependencies are out of date
+  * Limited to a whitelist of dependencies, doesn't cover transitive
+  * Only shows the dependencies from master branch
 
-* Library data from a slug
+* Dependencies included in a Slug
   * Covers all services that generate a slug
-  * Includes all dependencies used by service, including transitive
+  * Includes all runtime dependencies used by service, including transitive
   * Shows dependencies across different releases of slug, not just latest
 
+* The latest version for a whitelist of dependencies
```

Comment:
I initially read this as the update of the curated list itself, maybe small tweak to something like _The Latest version found for any of the whitelisted dependencies_ ?